AirCasting/SessionsSynchronization/Controller/SessionSynchronizationController.swift

```diff
@@ -6,18 +6,20 @@ import Combine
 import CoreLocation
 
 final class SessionSynchronizationController: SessionSynchronizer {
+    
     /// A plugin point for error handlers
     var errorStream: SessionSynchronizerErrorStream?
     
     private let synchronizationContextProvider: SessionSynchronizationContextProvidable
     private let downstream: SessionDownstream
     private let upstream: SessionUpstream
     private let store: SessionSynchronizationStore
-    private let dataConverter = SynchronizationDataConterter()
+    private let dataConverter = SynchronizationDataConverter()
+    private let viewModel: SessionSynchronizationViewModel
```

Comment:
view model cannot be used by the controller object, it's not how it's supposed to work. This class should be totally independent from other layers of the code like UI or the networking